prac_05/word_occurrences.py

```diff
@@ -0,0 +1,15 @@
+"""
+Word occurrences
+Estimate: 15 mins
+Actual: 20 mins
+"""
+from operator import *
```

Comment:
I can't see where you have used this imported module anywhere, probably the best idea to remove this